content/docs/user-guide/how-to/undo-adding-data.md

```diff
@@ -0,0 +1,46 @@
+# Undo Adding Data
+
+There are situations where you `dvc add` a data file and want DVC to stop
+tracking that file. Follow the steps listed here to undo `dvc add`.
+
+Let's first add a data file into an example <abbr>project</abbr> using
+`dvc add`, which creates a `.dvc` file to track the added data:
```

Comment:
Greatly summarized, thanks! i just have this suggestion to remove a small redundancy:
```suggestion
Let's first add a data file into an example <abbr>project</abbr> using
`dvc add`, which creates a `.dvc` file to track the data:
```

Reply:
Committing this.